src/plugins/spark/features/ActuatorOffset/ActuatorOffsetWidget.vue

```diff
@@ -50,6 +55,12 @@ export default class ActuatorOffsetWidget extends BlockWidget {
     </q-card-title>
     <q-card-separator/>
     <q-alert v-if="warnings" type="warning" color="warn">{{ warnings }}</q-alert>
+    <q-alert
+      v-if="!block.data.enabled"
+      :actions="[{label:'Enable', handler: enable }]"
+      type="info"
+      color="info"
+    >Offsetting {{ block.data.targetId }} is disabled</q-alert>
```

Comment:
change to active speech.

Reply:
- This block will not change/influence/offset targetid
- This block is disabled. TargetId will not be offset.